.github/workflows/stale.yml

```diff
@@ -20,6 +20,6 @@ jobs:
         days-before-close: 5
         stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Please comment or update this issue or it will be closed in 5 days.'
         stale-issue-label: 'Submitter Attention Required'
-        exempt-issue-labels: 'internal, Fixed in next release, Bug: Confirmed'
+        exempt-issue-labels: 'Internal, Fixed in next release, Bug: Confirmed'
```

Comment:
It might not actually be case sensitive. Not sure. See here where the bot that creates the sync PRs is told to apply `internal`

https://github.com/pi-hole/pi-hole/blob/master/.github/workflows/sync-back-to-dev.yml#L26

But it could also be that is part of that particular action, and this one requires case sensitivity...

Reply:
Looks like, in this case, case sensitivity should not be an issue:

https://github.com/actions/stale/pull/517